src/blocks/recommendations/style.scss

```diff
@@ -0,0 +1,18 @@
+.parsely-recommendations-list-title {
+	font-size: 1.2em;
+}
+
+.parsely-recommendations-ul {
+	list-style: none;
+}
+
+.parsely-recommendations-cardbody {
+	overflow: hidden;
+	padding: 0.8em;
+	text-overflow: ellipsis;
+	white-space: nowrap;
+}
+
+.parsely-recommendations-cardmedia {
+	padding: 0.8em 0.8em 0 0.8em;
```

Comment:
```suggestion
	padding: 0.8em 0.8em 0;
```

If the last positional item is removed, then it uses the value from the 2nd positional item.

Reply:
TIL - though we could leave this as is for readability/simplicity?